### PR TITLE
Update readme file of quantum Chinese chess

### PR DESCRIPTION
**Pull request by madcpf**
Update readme file of quantum Chinese chess to include game rules and commands.

### CODE REVIEW

File: examples/quantum_chinese_chess/README.md [hunk 1]
```diff
@@ -1,7 +1,107 @@
-# How to use the game
+Quantum Chinese Chess
+================
+This project is inspired by the [Quantum Chess project](https://quantumai.google/cirq/experiments/unitary/quantum_chess/concepts). It aims to introduce quantum moves to Chinese Chess (also called Xiangqi) so that the basic idea of quantum physics could be introduced to the players.
```
Comment (dstrain115) [inline examples/quantum_chinese_chess/README.md:3]: nit, I would probably keep the line length to 100 characters.

File: examples/quantum_chinese_chess/README.md [hunk 2]
```diff
@@ -1,7 +1,107 @@
...
 
-After cloning the Unitary library you can use command line flags to run the game:
+## How to run the game
+
+After cloning the Unitary library you can use the following command line to run the game:
```
Comment (dstrain115) [inline examples/quantum_chinese_chess/README.md:7]: nit: comma after library.

File: examples/quantum_chinese_chess/README.md [hunk 3]
```diff
@@ -1,7 +1,107 @@
...
   
 ```
 python -m examples.quantum_chinese_chess.chess
 ```
+
+Currently we don't have a graphical interface. Text display is supported at Linux terminal, Mac terminal, Colab, and Sublime Terminus.
```
Comment (dstrain115) [inline examples/quantum_chinese_chess/README.md:13]: comma after Currently

**Review by dstrain115 — APPROVED** [review]
Cool!  Love the documentation!

**Comment by madcpf** [comment]
> Cool! Love the documentation!

Thanks a lot for the review!

**Pull request merged**